### DISCUSSION TITLE
Office 365 modern SMTP connection

### DISCUSSION THREAD

**Discussion by kamcio2603** [YetiForceCompany/YetiForceCRM#14773] [Feature improvement]
Hello,

YetiCRM is using phpmailer which supports modern (aka SASL XOAUTH2 ) authentication for IMAP/SMTP email handling.

So, my question is, can CRM use/connect to office 365 via XOAUTH instead of old style login/pass.

Example link for google: https://phppot.com/php/sending-email-using-phpmailer-with-gmail-xoauth2/

**Comment by mariuszkrzaczkowski** [comment]
can use
only someone has to add the ability to CRM to be able to download the keys in the panel

**Reply by mariuszkrzaczkowski** [reply]
there are many possibilities only one problem is "who will do it"
